openlr/openlr_match_quality/openlr_assessment_tool/traffic_panel.cpp

```diff
@@ -69,7 +69,7 @@ void TrafficPanel::CreateTable(QAbstractItemModel * trafficModel)
   m_table->setSelectionBehavior(QAbstractItemView::SelectionBehavior::SelectRows);
   m_table->setSelectionMode(QAbstractItemView::SelectionMode::SingleSelection);
   m_table->verticalHeader()->setVisible(false);
-  m_table->horizontalHeader()->setVisible(false);
+  m_table->horizontalHeader()->setVisible(true);
```

Comment:
inline comment please

Reply:
Are you sure we need it? The name of the method is like an inline comment, isn't it?
`m_table->verticalHeader()->setVisible(...);`, `m_table->horizontalHeader()->setVisible(...);`, `m_table->setShowGrid(...);`

Reply:
indeed